internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,59 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var (
+	ErrorWrongFormat = errors.New("Неверный формат данных")
+	ErrorWrongSteps  = errors.New("Неверное количество шагов")
+	ErrorWrongTime   = errors.New("Неверный формат времени")
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
-}
+	dataSlice := strings.Split(datastring, ",")
+	if len(dataSlice) != 2 {
+		return ErrorWrongFormat
+	}
+	steps, err := strconv.Atoi(dataSlice[0])
+	if err != nil {
+		return err
```

Comment:
Лучше оборачивать ошибки
```suggestion
		return fmt.Errorf("invalid steps: %w", err)
```